meta/classes/nfsroot.bbclass

```diff
@@ -3,12 +3,12 @@ python nfsroot_eventhandler() {
     if d.getVar('NFS_UPDATER_INTERNAL'):
         return
 
-    # Set the path for the Unix socket
-    socket_path = '/tmp/nfsup.sock'
-
     import socket
     import os
 
+    # Set the path for the Unix socket
+    socket_path = f"/tmp/nfsup-{os.environ['USER']}.sock"
```

Comment:
This fails when run in a `kas` sandbox, as `USER` is not set in this environment. But I don't think we care about the username specifically here, so switching to `os.getuid()` should be fine.

Reply:
This is addressed by https://github.com/ejoerns/poky-nfsroot/pull/5 now.